lib/index.js

```diff
@@ -3,6 +3,7 @@
 // This file is licensed under the Artistic License 2.0.
 // License text available at https://opensource.org/licenses/Artistic-2.0
 
+'use strict';
```

Comment:
I think we should keep the `use stricts` at the top of the file (ie. the first line under the copyright?)